src/main/js/electron/index.py

```diff
@@ -1,79 +1,97 @@
+
 from flask import Flask, jsonify, render_template, request
 app = Flask(__name__)
 import json
 import plotly
-import plotly.graph_objs as go
+import plotly.offline as plt
 
 import pandas as pd
 import numpy as np
 import pkg_resources   
 
-@app.route('/')
-def my_form():
-    return render_template("welcome.html")
+@app.route('/hello')
+def hello():
+    return render_template('basic__plot.html')
 
 @app.route('/', methods=['POST'])
 def my_form_post():
-
+    print("Hi ")
     text = request.form['text']
+    start_time = request.form['from_time']
+    end_time =request.form['to_time']
+    start_count = request.form['from_count']
+    end_count =request.form['to_count']
+    
     if not text:
       return render_template("welcome.html");
+
     text="datasets/"+text
     light_curve = pkg_resources.resource_stream(__name__,text)
     data = np.loadtxt(light_curve)
     Time = data[0:len(data),0]
     Rate = data[0:len(data),1]
-    Error= data[0:len(data),2 ]
-    error = np.array(Rate) - np.array(Rate)+50
-    Final_error = np.array(error).tolist();
-    #print(Final_error)
-    graphs = [     
-      dict(     
-          data=[
-              go.Scatter(
-               x=Time,
-               y=Rate,
-               error_y=dict(
+    Error= data[0:len(data),2]
+    
+
+    import sys
+    import os
+    from plotly import session, tools, utils
+    import uuid
+    import json
```

Comment:
Imports at the top of the file please.
We don't ave to be religious about it but in general it would be good to follow "PEP 8": https://www.python.org/dev/peps/pep-0008/